dev-python/python-axolotl/python-axolotl-0.1.42.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="A python module for the axolotl protocol"
+HOMEPAGE="https://github.com/tgalal/python-axolotl"
+SRC_URI="https://github.com/tgalal/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
```
>>> Emerging (4 of 4) dev-python/python-axolotl-0.1.42::gentoo
>>> Downloading 'https://github.com/tgalal//archive/0.1.42.tar.gz'
--2018-06-23 18:10:46--  https://github.com/tgalal//archive/0.1.42.tar.gz
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-06-23 18:10:47 ERROR 404: Not Found.

Reply:
*grml*, sorry, should be now fixed.
Is there some debug switch, to force always a new download for SRC_URI during testing and ignoring local download?

Reply:
I use `export DISTDIR=/tmp/dist` while testing ;-).

Reply:
```
!!! Fetched file: python-axolotl-0.1.42.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      53869
!!! Expected: 39472

Reply:
* python-axolotl-0.1.42.tar.gz BLAKE2B SHA512 size ;-) ...                                                                                                                                                                        [ ok ]
>>> Unpacking source...
>>> Unpacking python-axolotl-0.1.42.tar.gz to /var/tmp/portage/dev-python/python-axolotl-0.1.42/work
>>> Source unpacked in /var/tmp/portage/dev-python/python-axolotl-0.1.42/work
>>> Preparing source in /var/tmp/portage/dev-python/python-axolotl-0.1.42/work/python-axolotl-0.1.42 ...

Emerging works here fine? :/ Using the Manifest from my commit here.

Reply:
Ah. I see the problem. The `python-axolotl-0.1.42.tar.gz` on the gentoo mirrors is faulty.. It differs from the one on Github upstream. Pushed correct manifest.